src/images/Cocktail.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="10" height="14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Icon images should all be 16x16. You can put this icon in a 16x16 frame in Figma, then re-export the frame as an SVG to resolve.